OracleGoldenGate/Dockerfile

```diff
@@ -1,6 +1,32 @@
-# LICENSE CDDL 1.0 + GPL 2.0
+# Copyright (c) 2017 Oracle and/or its affiliates.  All rights reserved.
 #
-# Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
+# The Universal Permissive License (UPL), Version 1.0
```

Comment:
There is no need to paste the entire License here. Just say it is licensed under UPL 1.0